src/screens/admin/AdminDashboard.tsx

```
@@ -0,0 +1,64 @@
+	TabList,
+	TabPanels,
+	Tab,
+	TabPanel,
+} from '@chakra-ui/react';
+
+import FieldMappingTab from '../../components/FieldMappingTab';
+import DocumentConfigurationTab from '../../components/DocumentConfigurationTab';
+
+const AdminDashboard = () => {
+	// State to track which tab is active and trigger refresh
+	const [activeTab, setActiveTab] = useState(0);
```

ESLint

[error] 'activeTab' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)